definitions/release/install.yaml

```diff
@@ -2228,6 +2228,7 @@ rules:
   - configmaps
   - resourcequotas
   - limitranges
+  - roles
```

Comment:
Let's remove this change from release/install.yaml for now, we'll update this manifest every time we do a release. The next release v1.3.5 should be available in a few days.

Reply:
Got it, removed the line.